runtime/moonriver/tests/integration_test.rs

```diff
@@ -462,7 +462,7 @@ fn verify_pallet_indices() {
 	is_pallet_index::<moonriver_runtime::Migrations>(34);
 	is_pallet_index::<moonriver_runtime::ProxyGenesisCompanion>(35);
 	// Ethereum compatibility
-	is_pallet_index::<moonriver_runtime::EthereumChainId>(50);
+	is_pallet_index::<moonriver_runtime::EvmChainId>(50);
```

Comment:
We usually try not to reuse pallet indices like this, but it might be ok if the pallets are identical. I think the safer choice is to use a fresh index (`53`?) though.

Reply:
Used 54 as 53 was already used by on old pallet.

Reply:
Yes always use a different id for a different pallet, the main reason is to avoid incompatibility, mostly due to the fact that the extrinsics use the index to specify the pallet call.

Reply:
I really think we should use the same pallet index (50). The pallet is identical, so I dont think there is a reason not to re-use it. It also has no extrinsics whatsoever, so there is not really a risk I think..

Reply:
There's disagreement in the comments here, so I unresolved.

@girazoki do you see any harm in using a fresh pallet index? I agree with your assessment, but also don't see any harm in using a new one.

Reply:
not really, using the same one or different one only affects things like XCM (where you rely on extrinsic indices to perform transactions). So I dont see a risk neither using the same nor using a different one